_posts/2015-09-16-introduction-of-recommender-algorithms-for-e-commerce.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Introduction of Recommender Algorithms for E-Commerce"
+date: "2015-09-16"
+icon: Light-bulb
+categories: tech-stories
+authors: ["Pujit"]
+---
+
+##Introduction
+
+Recommender systems in e-commerce can be seen as a sales personal.  Like a good sales personal in a store helps increasing sales and building loyalty customer base, recommender system handles sales three ways: browser into buyers, cross-sell and loyalty.
```

Comment:
@maletzke @steffimeier 
1. I accept mistake on singular/plural case for recommender system/s in first paragraph of introduction.  Also I am removing empty spaces.
2. Basic knowledge of Recommender Systems in Computing Space is prerequisite for understanding this article.
3. Terminology(including 'browser into buyers'), used in this article is coming from published scientific research papers. Please have a look into references.